be-java-springboot/files/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk/openjdk11:alpine-jre
+FROM registry.access.redhat.com/ubi8/openjdk-17
```

Comment:
switching to redhat ubi8 image

Reply:
Like it, thank you

Reply:
Note https://www.redhat.com/en/blog/transitioning-red-hat-container-registry. TBH I am not really sure what we should do about it. We can use `registry.redhat.io` instead, but that would break local builds. Another approach is to replace the `FROM` on the fly (via the `BuildConfig`) but that might need some ground work first.

Reply:
this doesn't sound good. I think it would be better to stick with adoptopenjdk until the red hat registry issue is being clarified.
I'll change the image to `eclipse-temurin:17-jdk-alpine` which is and uses the java adoptium distribution from Adoptium, which is the official and equivalent successor of adoptopenjdk (see: https://adoptium.net/)

Reply:
@michaelsauter we need your feedback.... can you please comment on this?

Reply:
@michaelsauter thanks for the feedback. No I haven't read about an activation date of oauth from red hat. Since this discussion is still open and not fully discussed I think it is fine to leave at it is. We can change the docker image later on to a unified one.